roles/openshift_metering/files/metering-helm-operator-deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: metering-helm-operator
+  labels:
+    app: metering-helm-operator
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: metering-helm-operator
+  template:
+    metadata:
+      labels:
+        app: metering-helm-operator
+    spec:
+      securityContext:
```

Comment:
This almost always causes trouble and probably needs removed; the platform will default it appropriately.

Reply:
This is well tested against openshift it shouldn't have issues unless it was set to false.

Reply:
Tested in an Online cluster (e.g. free-int)? The GCP clusters you get from the CI tooling don't have the same security policy setup...

Reply:
That's...going to make validating things difficult. This PR is the pre-req for trying to deploy into online, so not there yet. Where can I get a copy of the that those settings so I can validate that this works?

Reply:
Confirmed in slack this shouldn't be an issue.